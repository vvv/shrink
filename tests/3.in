a
b
c
c
c
d TEST 1
d TEST 2
d TEST 100
d TEST 100
e
f
f
g
h TEST 4
i TEST 5
j TEST 6
k TEST' 7
l TEST' 8
m TEST' 9
n TEST' 10
