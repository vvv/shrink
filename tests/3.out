a
b
c
.. Last line repeated 3 times
d TEST 1
.. 2 more lines are pretty similar
d TEST 100
e
f
f
g
h TEST 4
.. 2 more lines are pretty similar
k TEST' 7
.. 3 more lines are pretty similar
